config.py

```diff
@@ -33,6 +34,10 @@ def __init__(self):
         self.create_xray_when_sending.setChecked(prefs['create_xray_when_sending'])
         self.l.addWidget(self.create_xray_when_sending)
 
+        self.expand_aliases = QCheckBox('Auto associate split aliases')
```

Comment:
Right, that's why I named it the way it is. I updated the readme to explain it. looking into QWhatsThis now.